src/workspace/workspace.es6

```diff
@@ -55,13 +55,13 @@ const state = {
       save_name: w => state.workspace.perv_name = sanitize(w.name),
       blur: el => el.blur(),
       rename: w => {
-        const perv_name = state.workspace.perv_name;
+        const perv_name = sanitize(state.workspace.perv_name);
```

Comment:
We might as well update this to `prev_name` also since we are changing it + the other `pervs` also 😉

Reply:
there are quite number of `pervs` in other files too. for now i only change `pervs` in workspace.